app/requirements.txt

```diff
@@ -4,6 +4,7 @@ django-filter==0.11.0
 django-extensions==1.6.1
 django-cors-headers==1.0.0
 google-api-python-client==1.5.0
+oauth2client==2.0.1
```

Comment:
I also just saw the same thing when checking out develop and provisioning (twice). oauth2client import problem that was causing my queries to fail.